docs/source/models.rst

```diff
@@ -711,13 +704,14 @@ TensorFlow (``tensorflow``)
 
 The ``tensorflow`` model flavor allows serialized TensorFlow models in
 `SavedModel format <https://www.tensorflow.org/guide/saved_model#save_and_restore_models>`_
+or Keras models in MLflow Model format
```

Comment:
Mentioning both `TensorFlow models` and `Keras models` is confusing. Can we remove `Keras models`?

Reply:
In Tensorflow 2.0, is `TensorFlow models` equivalent to `Keras model`?